netlify.toml

```diff
@@ -4,4 +4,5 @@
   functions = ".netlify/functions/"
 
 [build.environment]
-  NPM_FLAGS="--prefix=/dev/null"
\ No newline at end of file
+  NPM_FLAGS="--prefix=/dev/null"
+  NODE_VERSION="12.22"
```

Comment:
either is fine, but I might use a newer version like `14.16.0`

Reply:
Yeah, lets try that.